ci-operator/step-registry/mirror-images/by-oc-image-mirror/mirror-images-by-oc-image-mirror-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: mirror-images-by-oc-image-mirror
```

Comment:
As discussed offline, the step name `mirror-images-qe-test-images` is better, to reflect its function.